dev/compose/opensearch/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM opensearchproject/opensearch:2.12.0
+
+RUN opensearch-plugin remove opensearch-skills --purge
+RUN opensearch-plugin remove opensearch-ml --purge
+RUN opensearch-plugin remove opensearch-neural-search --purge
+RUN opensearch-plugin remove opensearch-performance-analyzer --purge
+RUN opensearch-plugin remove opensearch-security-analytics --purge
```

Comment:
This culls a bunch of pointless plugins (for local dev) that just slow startup times down a ton.